### DISCUSSION TITLE
Is this crate appropriate as a single threaded ws client?

### DISCUSSION THREAD

**Discussion by RohanKapurDEV** [Gelbpunkt/tokio-websockets#33] [Q&A] [answered]
Does it make sense to use this crate as a websocket client in a situation where the tokio runtime is only running on the current thread? Or is this not meant to be used in a single-threaded context?

**Comment by Gelbpunkt** [comment, marked as answer]
It makes perfect sense in both single threaded and multi threaded runtime contexts.
